tests/unittests/test_util.py

```diff
@@ -1571,6 +1571,46 @@ def test_already_mounted_calls_callback_with_data(
             mock.call(mock.ANY, mock.sentinel.data)
         ] == callback.call_args_list
 
+    @mock.patch(M_PATH + "subp.subp")
+    def test_mount_cb_does_not_log(self, m_subp, caplog):
+        log_msg = (
+            "Failed to mount device: '/dev/sda1' with type: "
+            "'ntfs' using mount command:"
+        )
+        m_subp.side_effect = subp.ProcessExecutionError(
+            "", "unknown filesystem type 'ntfs'"
+        )
+        callback = mock.Mock(autospec=True)
+        with pytest.raises(Exception):
+            util.mount_cb(
+                "/dev/sda1",
```

Comment:
the tests may be failing if the system mounts are leaking and /dev/sda1 is already mounted?